streamwebs_frontend/streamwebs/migrations/0002_auto_20170720_1641.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-07-20 16:41
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('streamwebs', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='soil_survey',
+            name='site_char',
+            field=models.TextField(blank=True, verbose_name='distinguishing site characteristics'),
+        ),
+    ]
```

Comment:
Sadly, changing the verbose name does in fact generate a new migration. It's trivial and petty, but changing the name to remove whitespace make the translation string cleaner.